File: suco/suco/src/main/java/com/example/suco/dto/sos/tinhieu/SOSMapResponseDTO.java
package com.example.suco.dto.sos.tinhieu;

public class SOSMapResponseDTO {

    private Long id;
    private Double viDo;
    private Double kinhDo;
    private Long truSoId;
    public SOSMapResponseDTO() {}
    public SOSMapResponseDTO(Long id, Double viDo, Double kinhDo, Long truSoId) {
        this.id = id;
        this.viDo = viDo;
        this.kinhDo = kinhDo;
        this.truSoId = truSoId;
    }
    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }
    public Double getViDo() { return viDo; }
    public void setViDo(Double viDo) { this.viDo = viDo; }
    public Double getKinhDo() { return kinhDo; }
    public void setKinhDo(Double kinhDo) { this.kinhDo = kinhDo; }
    public Long getTruSoId() { return truSoId; }
    public void setTruSoId(Long truSoId) { this.truSoId = truSoId; }

    
}

File: CanhBao/app/src/main/java/com/example/canhbao/data/model/sos/tinhieu/SOSMapResponseDTO.kt
package com.example.canhbao.data.model.sos.tinhieu

import com.example.canhbao.data.model.info.truso.TruSoMiniDTO

data class SOSMapResponseDTO(
    val id: Long?,
    val viDo: Double?,
    val kinhDo: Double?,
    val trangThai: String,
    val truSo: TruSoMiniDTO? 
)